docs/source/upcoming_release_notes/742-Fix_3rd_Att_Transmission_Status.rst

```diff
@@ -0,0 +1,30 @@
+742 Fix 3rd Att Transmission Status
+#################
+
+API Changes
+-----------
+- N/A
+
+Features
+--------
+- N/A
+
+Device Updates
+--------------
+- N/A
+
+New Devices
+-----------
+- N/A
+
+Bugfixes
+--------
+- The transmission status value for the 3rd harmonic has been fixed, it was previously using the wrong value.
+
+Maintenance
+-----------
+- N/A
+
+Contributors
+------------
+- N/A
```

Comment:
```suggestion
- cristinasewell
```

Reply:
Thanks Zach I forgot about this - changed this in 2d6d7ad